src/helpers/displayErrorMessagesAddUser.js

```diff
@@ -0,0 +1,23 @@
+import { errors } from '../helpers/errorMessages'
+import { errorHandle } from './errorMessageHandle'
+
+export const displayErrorMessagesAddUSer = (allFieldsFilledIn, emailValidation, phoneValidation, doubleUserValidation, doubleUserValidationInput, dispatchGetFormError, formData) => {
+
+  const emptyField = allFieldsFilledIn === false ? errors.errorNotAllFieldsAreFilledIn : null
```

Comment:
Nie stosuj sprawdzania czy dany warunek nie został spełniony:
```
if (false === isEnabled) {...}
```
Tak przebuduj funkcję, żeby sprawdzać, czy warunek został spełniony:
```
if (isDisabled) {...}
```
Taka forma jest dużo bardziej czytelna dla innych programistów. Przypomnij sobie monolog Pinokia ze Shreka: "nie mogę z całą pewnością stwierdzić gdzie jest, ale wiem na pewno, gdzie go nie ma..." - to przypomina czytanie kodu, który opiera się na sprawdzaniu czy dany warunek nie został spełniony :)